e2e/specs/multipage_apps.spec.js

```diff
@@ -56,4 +56,21 @@ describe("st.map", () => {
 
     cy.get(".element-container .stMarkdown p").should("contain", "x is 1");
   });
+
+  it("can switch to the first page with a duplicate name", () => {
+    cy.getIndexed('[data-testid="stSidebarNav"] a', 3).click();
+
+    cy.get(".element-container .stMarkdown h2").should("contain", "Page 4");
+  });
+
+  it("can switch to the first page with a duplicate name", () => {
```

Comment:
should this be switch to the second page?

Reply:
Oh whoops, yeah -- fixed